core-java-modules/core-java-numbers-6/src/test/java/com/baeldung/javadoublevsbigdecimal/BigDecimalUnitTest.java

```diff
@@ -0,0 +1,50 @@
+package com.baeldung.javadoublevsbigdecimal;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BigDecimalUnitTest {
+    private BigDecimal bd1;
+    private BigDecimal bd2;
+    private BigDecimal expected;
+    private BigDecimal actual;
```

Comment:
Since expected and actual are created or reassigned in every test we can create just local variables instead.